external_dns/assets/configuration/spec.yaml

```diff
@@ -0,0 +1,42 @@
+name: External DNS
```

Comment:
Was this file change accidentally included?

Reply:
yes